css/ui/style.css

```diff
@@ -124,19 +119,6 @@
     overflow-x: hidden;
 }
 
-.scratchpad-ace-editor .ace_sb {
```

Comment:
"ace_sb" also no longer seems to select anything, and I couldn't find it in the ace codebase.